mega-zep-backend/src/test/java/com/gepardec/mega/zep/service/impl/WorkerServiceImplTest.java

```diff
@@ -108,4 +103,42 @@ void testGetMonthendReportForUser_MitarbeiterValid_ProjektzeitenValid_Warning()
                         Task.BEARBEITEN,
                         WorkingLocation.DEFAULT_WORKING_LOCATION));
     }
+
+    private Employee createEmployee(final int userId) {
```

Comment:
this method has the same code as createEmployeeWithReleaseDate, overloaded method could be used

Reply:
Lets do this in the next iteration, I want to merge this, because another heavy re-factoring is on the way